### ISSUE TITLE
Training bookdown for ADC

### ISSUE THREAD

**Issue by vlraymond** [NCEAS/arctic-data-outreach#24]
bookdown - https://www.rstudio.com/resources/webinars/introducing-bookdown/

Can we do gitbook? 
https://www.gitbook.com/editor

**Comment by mbjones** [comment]
What would the advantages of that be?  The nice thing about bookdown is that it is very easy to embed and execute R code as part of the tutorial docs.  And there's really nice preview support in RStudio.  Does gitbook support RMarkdown files, or just Markdown files?

**Comment by vlraymond** [comment]
Yes, you're right Matt. I watched a tutorial on Bookdown on Friday and came to the same conclusion.

**Comment by vlraymond** [comment]
[bookdown-demo.pdf](https://github.com/NCEAS/arctic-data-outreach/files/1752748/bookdown-demo.pdf)

**Comment by vlraymond** [comment]
^^^ draft as of 2/23/18.

Process so far:
combing through existing content on training.arcticdata.io
copying, combining, collating in to content for the book

next steps:
refining and expanding existing content
creating new content

**Comment by vlraymond** [comment]
[bookdown-demo.pdf](https://github.com/NCEAS/arctic-data-outreach/files/1753105/bookdown-demo.pdf)
latest

**Comment by vlraymond** [comment]
determine:

- intended audience
- R users / technical users

**Comment by vlraymond** [comment]
other bookdowns in our NCEAS ecosystem that could be good resources
- https://nceas.github.io/datateam-training/using-git-in-rstudio.html
- https://nceas.github.io/sasap-training/materials/reproducible-analysis-in-r/

**Comment by vlraymond** [comment]
general guidelines:
- short (no longer than 1 hr of content)
- make things general / generic when possible


known issues:
- how to manage content-specific datasets   for training examples

**Comment by vlraymond** [comment]
next step: VR needs to finally push this to the repo
